scripts/run_sim.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+sonata_simulator -E $SONATA_SIM_BOOT_STUB -E $1
```

Comment:
Where is `SONATA_SIM_BOOT_STUB` defined?

Reply:
In the nix environment: https://github.com/lowRISC/sonata-software/blob/d8ba1e1b74a6e3bf1ed92e4bd89e2c9d7f1550bb/flake.nix#L40

A check to ensure it is defined is a good idea.

Reply:
Ah cool.

Reply:
It's in the nix devShell